app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def create
+    @article = Article.find(params[:article_id])
```

Comment:
Should strong params be used throughout the whole method if they're being used at all?